Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,25 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    unless Dir.glob("#{formula.prefix}/**/*").any? do |f|
```

Comment:
Assuming there is a missing `end` somewhere that went unnoticed, I think that shows how confusing it is to write it this way.

Let's not use unless + a multi-line condition. Why not just iterate over the collection and return from the method if you find a file?